components/pagination/stories/template.js

```diff
@@ -99,21 +99,32 @@ export const Template = ({
 			}, context)}
 		</nav>
 	`;
+
+	if (variant === "explicit") {
+		return explicitVariant;
+	}
+	else if (variant == "button") {
+		return buttonVariant;
+	}
+	return listingVariant;
 };
 
 export const PaginationGroup = (args, context) => {
 	return html`
 		<div style=${styleMap({
-			"display": window.isChromatic() ? "none" : undefined,
+			"display": window.isChromatic() && context.viewMode !== "docs" ? "none" : undefined,
```

Comment:
Depending on what gets merged first, and since we already have some of this Chromatic preview stuff in place, did we want to refactor this `PaginationGroup` to use `Variants()`? 

Looks like it's being addressed in[ #2912 ](https://github.com/adobe/spectrum-css/pull/2912/files#diff-ae92bc343a4abb638a6bfdc1168db6fcab504115107fb5f7891062510c4acce5), but we might want to keep that though in our back pocket if this PR looks like it'll get merged first. 🤷‍♀️ How would you handle it?

Reply:
Thanks for pointing that out in that upcoming PR. I'll have to address the rebase conflicts when it merges.